arbstate/inbox.go

```diff
@@ -122,7 +122,13 @@ func parseSequencerMessage(ctx context.Context, batchNum uint64, data []byte, da
 			log.Warn("sequencer msg decompression failed", "err", err)
 		}
 	} else {
-		log.Warn("unknown sequencer message format")
+		length := len(payload)
+		var firstByte byte
+		if length > 0 {
+			firstByte = payload[0]
+		}
+		log.Warn("unknown sequencer message format", "length", length, "firstByte", firstByte)
```

Comment:
Maybe we want a different error for any empty payload since otherwise it looks like the 0 byte

Reply:
:+1: